src/lib/components/dashboard/groups/AddUserToGroup.svelte

```diff
@@ -5,32 +5,22 @@
         fetchAllUsers,
         addUserToGroup,
         fetchCredentialFieldsByGroupId,
+        fetchUsersWithoutGroupAccess,
     } from "../apis";
     import { CredentialFields } from "../dtos";
     import { createShareCredsPayload } from "../helper";
 
     import { User } from "../dtos";
     import { selectedGroup, showAddUserToGroupDrawer } from "../store";
+
     let users: User[] = [];
     let credentialFields: CredentialFields[] = [];
     onMount(async () => {
         if (!$selectedGroup) return;
-        const userGroupPromise = fetchGroupUsers($selectedGroup.groupId);
-        const allUsersPromise = fetchAllUsers();
-        const [userGroup, allUsers] = await Promise.all([
-            userGroupPromise,
-            allUsersPromise,
-        ]);
-        // TODO: API to fetch non group members.
-        users = allUsers.data.filter(
-            (user) =>
-                !userGroup.data.some((groupUser) => groupUser.id === user.id),
-        );
-
-        const credentialFieldsResponse = await fetchCredentialFieldsByGroupId(
```

Comment:
This an important part of the code. why did you remove this?

Reply:
sorry missed it